Gulpfile.js

```diff
@@ -147,6 +155,43 @@ var PUBLISH_TAG = JSON.parse(fs.readFileSync(path.join(__dirname, '.publishrc'))
 
 var websiteServer = null;
 
+gulp.task('audit', function (done) {
+    let npmCommand = 'npm';
```

Comment:
I propose to start a clean up process for the `gulpfile.js` as mention in https://github.com/DevExpress/testcafe/pull/919.

Create a file `gulp-tasks/npm-audit.js`, move this code in it and run task as following:
```js
const npmAudit = require('gulp-tasks/npm-audit.js');
...

gulp.task('audit', () => {
    // returns Promise
    return npmAudit.perform();
});
```

Reply:
If it makes sense, for running npm commands you can use the [npm-utils](https://www.npmjs.com/package/npm-utils), [npm-audit](https://www.npmjs.com/package/npm-audit-report) or something like that

Reply:
I agree we need extract this code to a separate file, but I am against to refactor `Gulpfile.js` on dozens of small files now.

Reply:
Ok. Let it be